pkg/scheduler/scheduler.go

```diff
@@ -600,7 +581,7 @@ func (sched *Scheduler) scheduleOne() {
 	// Synchronously attempt to find a fit for the pod.
 	start := time.Now()
 	state := framework.NewCycleState()
-	scheduleResult, err := sched.schedule(pod, state)
+	scheduleResult, err := sched.schedule(podInfo, state)
```

Comment:
should we move recordSchedulingFailure from inside schedule to here to make it consistent with the reset of cases?

Reply:
SG. After that, I also removed the `schedule` function since it doesn't do much any more.